test/e2e/common/junit/junit.go

```diff
@@ -0,0 +1,11 @@
+package junit
+
+import (
+	"os"
+)
+
+func ConstructJunitFileName(name string) string {
+	reportDir := os.Getenv("e2e_reports_dir")
+	testGroupPrefix := "e2e."
+	return reportDir + "/" + testGroupPrefix + name
```

Comment:
I second @blaisedias 's comment and would return fully constructed `[]Reporters` here. More DRY, better maintainability going forward.

Reply:
Now done.